internal/drivers/sqlserver/sql.go

```diff
@@ -203,7 +203,7 @@ func createSQL(s *internal.Schema) string {
 
 func parseURLToDSN(urlstr string) (string, error) {
 	// Example input: "sqlserver://sa:eds@localhost:11433/database=eds"
-	// Desired output: "sqlserver://sa:eds@localhost:11433/database=eds&multiStatements=true"
+	// Desired output: "sqlserver://sa:eds@localhost:11433/database=eds?multiStatements=true"
```

Comment:
i don't like `database=<name>` as this is too different than the others... since we're parsing this ourselves i suggest we make it this:


```
sqlserver://sa:eds@localhost:11433/eds?multiStatements=true
```

and if SQLServer expects that weird format we can transform into that.

Reply:
I checked and doing it in this format works without any changes needed `sqlserver://sa:eds@localhost:11433/eds?multiStatements=true`